python/tidy/servo_tidy/tidy.py

```diff
@@ -1052,16 +1052,24 @@ def run_lint_scripts(only_changed_files=False, progress=True, stylo=False):
 
 
 def check_commits(path='.'):
+    """Checks if the test is being run under Travis CI environment"""
+    is_travis = (os.environ.get('TRAVIS') == 'true')
```

Comment:
Please add a comment before this line as to why we have to do this (that Travis gets a merge commit from Github, which should be ignored by tidy in Travis environment).